packages/MarkdownEditor-Core.package/MarkdownReadStream.class/methodProperties.json

```diff
@@ -0,0 +1,7 @@
+{
+	"class" : {
```

Comment:
I'm sorry, I can't comment on a empty file; didn't you want to add a comment to `MarkdownReadStream` something like "This is for compatibility" (and for stuff we always wanted to have when you apply my proposal)?

Reply:
We should probably add class comments to all of the classes. 🙃